src/composables/useColourScale.ts

```diff
@@ -13,7 +13,9 @@ interface IndicatorRange {
 }
 
 type FillAndOutlineColour = {
+    // the fill colour of the feature
     fillColor: string;
+    // the outline colour of the feature
     color: string;
 };
```

Comment:
My preference would be to rename `color` to `outlineColor`, as opposed to making people hunt for the inline comment. Sorry it's a bit tedious to rename.

Reply:
no worries! done